tests/README.md

```
@@ -1,105 +1,159 @@
+│   ├── serve/
+│   ├── kvbm/
+│   ├── benchmark/
+│   ├── fault_tolerance/
+│   └── ...
+```
+- Place **unit/integration tests** for a component in its `tests/` subfolder.
+- Place **end-to-end (E2E) tests** and cross-component tests in `dynamo/tests/`.
+- Name test files as `test_<component>_<flow>.py` for clarity.
+
+### Test Types and Locations
+| Type         | Description                              | Location              |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)